src/pipecat/services/sarvam/tts.py

```diff
@@ -109,24 +124,26 @@ def __init__(
             api_key: Sarvam AI API subscription key.
             voice_id: Speaker voice ID (e.g., "anushka", "meera"). Defaults to "anushka".
             model: TTS model to use ("bulbul:v1" or "bulbul:v2"). Defaults to "bulbul:v2".
-            aiohttp_session: Shared aiohttp session for making requests.
             base_url: Sarvam AI API base URL. Defaults to "https://api.sarvam.ai".
             sample_rate: Audio sample rate in Hz (8000, 16000, 22050, 24000). If None, uses default.
+            aiohttp_session: Optional shared aiohttp session for making requests.
+                If None, a new session will be created. Defaults to None.
             params: Additional voice and preprocessing parameters. If None, uses defaults.
             **kwargs: Additional arguments passed to parent TTSService.
         """
         super().__init__(sample_rate=sample_rate, **kwargs)
 
-        params = params or SarvamTTSService.InputParams()
+        params = params or SarvamHttpTTSService.InputParams()
 
         self._api_key = api_key
         self._base_url = base_url
-        self._session = aiohttp_session
+        self._session = aiohttp_session or aiohttp.ClientSession()
```

Comment:
Required after the change I suggested.

```suggestion
        self._session = aiohttp_session
```